temporian/proto/core.proto

```diff
@@ -157,7 +157,7 @@ message OperatorDef {
   }
 
   message Attribute {
-    // String identifier of the atrribute. Should be unique.
+    // String identifier of the atribute. Should be unique.
```

Comment:
typo `atribute` -> `attribute`

Reply:
hehe amazing to fix a typo and generate a new one 🤦